ledger/participant-integration-api/src/docs/resources/com/daml/platform/docs/self-service-error-codes-migration.yml

```diff
@@ -57,7 +46,7 @@ duplicateCommandException:
 ledgerIdMismatch:
   change:
     - NOT_FOUND
-    - NOT_FOUND
+    - FAILED_PRECONDITION
```

Comment:
No text?

Reply:
This change is actually stale (stemmed from a PR from yesterday) on which I backtracked. I've rebased this PR too many times and this stale guy went through. Good catch, I will actually revert this one in a following PR